### PR TITLE
Update README for release

### PR DESCRIPTION
**Pull request by vharsh**
Signed-off-by: Harsh Vardhan <harsh.vardhan@mayadata.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,33 +46,29 @@ The CLI currently supports managing `cStor`, `Jiva`, `LocalPV-LVM`, `LocalPV-ZFS
...
-   tar -xvf kubectl-openebs_v0.2.0_Linux_x86_64.tar.gz
-   cd kubectl-openebs_v0.2.0_Linux_x86_64
-   sudo mv kubectl-openebs /usr/local/bin/
-   ```
-  Or, download the `debian` package from the [release](https://github.com/openebs/openebsctl/releases) page and double click it launch the installer if using GUI.<br/><br/>
-  Or, we can also use script to install the latest version :-
-   ```shell
-   wget https://raw.githubusercontent.com/openebs/openebsctl/develop/scripts/install-latest.sh -O - | bash
-   ```
-* For Mac, download the respective tarball from [release](https://github.com/openebs/openebsctl/releases) page and :-
```
Comment (Abhinandan-Purkait) [inline README.md:65]: Do we want to remove the steps of installation for mac and windows?

File: README.md [hunk 2]
```diff
@@ -15,12 +15,12 @@
...
 
 
-### Project Status
+## Project Status
 
 **Alpha**. Under active development and seeking [contributions from the community](#contributing).
 The CLI currently supports managing `cStor`, `Jiva`, `LocalPV-LVM`, `LocalPV-ZFS` Cas-Engines.
 
-### Table of Contents
+## Table of Contents
```
Comment (Abhinandan-Purkait) [inline README.md:23]: Table of contents is not updated.

File: README.md [hunk 3]
```diff
@@ -302,6 +372,9 @@ OpenEBSCTL is available on Linux, macOS and Windows platforms.
     VERSION       : 2.1.0
     UPGRADING     : true
     ```
+  * #### Degbugging
+
```
Comment (Abhinandan-Purkait) [inline README.md]: Image doesnot seem to be loading and also the command is missing in image.

File: README.md (hunk 1 above)
Comment (vharsh) [reply README.md:65]: Yes, krew is the best way to download & update the plugin. Besides, once people use krew going to the releases page won't make sense.

Comment (Abhinandan-Purkait) [reply README.md:65]: Okay. Awesome

File: README.md [hunk 4]
```diff
@@ -302,6 +377,11 @@ OpenEBSCTL is available on Linux, macOS and Windows platforms.
     VERSION       : 2.1.0
     UPGRADING     : true
     ```
+  * #### Debugging pvcs
```
Comment (Abhinandan-Purkait) [inline README.md:380]: Can we add a note here, that this is supported only for `cStor`.

**Review by Abhinandan-Purkait — APPROVED** [review]

**Pull request merged**